go.mod

```diff
@@ -399,6 +400,7 @@ replace (
 	// github.com/stackrox/helm-operator is a modified fork of github.com/operator-framework/helm-operator-plugins that
 	// we currently depend on.
 	github.com/operator-framework/helm-operator-plugins => github.com/stackrox/helm-operator v0.0.12-0.20221003092512-fbf71229411f
+	github.com/stackrox/scanner => github.com/stackrox/scanner v0.0.0-20221031200236-942ec05c53b4
```

Comment:
no need for this replace anymore. You can just go to line 88 and set the version to `master` then run `go mod tidy`